packages/block-library/src/table/block.json

```diff
@@ -125,6 +122,11 @@
 	"supports": {
 		"anchor": true,
 		"align": true,
+		"color": {
+			"__experimentalSkipSerialization": true,
+			"gradients": true,
+			"link": false
```

Comment:
Probably not necessary at the moment as the existing `getColorAndStyleProps` doesn't process link color.

Reply:
I believe the intent of the current `getColorAndStyleProps` was to only bring in what was needed for the button block. That happened to line up with what was needed here for the table block. 

More than likely, a generic utility would need to take into account the current block's configuration of block supports.